packages/teams-js/src/public/interfaces.ts

```diff
@@ -1132,6 +1132,10 @@ export interface AppEligibilityInformation {
    * A user will be in at most one cohort.
    */
   cohort: Cohort | null;
+  /**
+   * Feature Sets
+   */
+  featureSet?: FeatureSet;
```

Comment:
>? [](http://example.com/codeflow?start=12&length=1)

What would be the semantic difference between having this property be `undefined` vs defined but with two empty array properties?

I'm guessing the answer is something like "this has to have the `?` because older version of the host won't know to send this property so it will show up as undefined then." If that's true, then consider adding documentation for the property that says:
"If this property is undefined, it indicates that the host is an older version that doesn't support this property."

Reply:
My understanding is it's the latter, so I think adding a comment is the way to go here.

Reply:
yep it is for back compatibility. Added the comment!